app/components/views/Offset/index.tsx

```diff
@@ -542,12 +639,9 @@ export const Offset = (props: Props) => {
                   !!beneficiaryAddress && !utils.isAddress(beneficiaryAddress)
                 }
                 onChange={(e) => handleBeneficiaryAddressChange(e.target.value)}
-                placeholder={t({
-                  id: "offset.enter_address",
-                  message: "Enter 0x address",
-                })}
+                placeholder={t`Beneficiary 0x address (optional)`}
```

Comment:
Not a blocker, just starting a conversation (:
Are we dropping translation ids?

Reply:
Ah this was me experimenting but I forgot to revert it.
But i've been ruminating on the idea of slowly migrating away from ids, except for maybe common strings. The main benefit of ids are:
1. make it easier to reuse strings (in theory-- in practice we probably have many duplicates)
2. make it easier for translators to find them in the ui, if we are good about writing helpful ids `pledge.edit_modal.loading`

Reply:
> Open question: if I remove a string id but keep the source string, does that trigger a re-translation?

I guess this would make a change in translation.io when an existing ID is removed.
And would be a new translation afterwards because internally lingui creates a new ID. 

See your own result in this PR [here](https://github.com/KlimaDAO/klimadao/pull/746/files#diff-c807d6bc34a846ae9029acc99e48d0bedf787ca01fe7e2194628976f5ba567d0R17-R23) 
=> a new ID from the message string was created

I would keep IDs for now.
Especially because detecting doubled translations is easier.